acceptance/ws_test/.kitchen.ec2.yml

```diff
@@ -0,0 +1,30 @@
+driver:
+  name: ec2
+  instance_type: c4.large
```

Comment:
💸

Reply:
Waiting also costs time.

Reply:
We should use c5 class instead of the c4 class.